content/reference/storage/api.md

```diff
@@ -227,22 +229,44 @@ Type can be any of:
 
 You can write and delete values from a Worker, but you should note that it is an eventually consistent data store. In practice, this means it is not uncommon for an edge location to continue returning an old value for up to a minute after that key is written in some other edge location. If, after considering that, it makes sense to write values from your Worker, the API is:
 
-`NAMESPACE.put(key, value)`
+{{<highlight typescript>}}
+NAMESPACE.put(key: string, value: string | ReadableStream | ArrayBuffer)
+{{</highlight>}}
 
 The type is automatically inferred from value, and can be any of:
 
-- string
-- ReadableStream
-- ArrayBuffer
+- `string`
+- `ReadableStream`
+- `ArrayBuffer`
 
```

Comment:
One thing I queried was the lack of `FormData` type. It shows up in [the actual TypeScript type declarations](https://github.com/cloudflare/workers-types/blob/master/index.d.ts#L168), but no where else that I could see.